lib/zendesk_api/middleware/request/retry.rb

```diff
@@ -8,7 +8,7 @@ module Request
       # @private
       class Retry < Faraday::Middleware
         DEFAULT_RETRY_AFTER = 10
-        ERROR_CODES = [429, 503]
+        ERROR_CODES = [429, (500..599).to_a].flatten
```

Comment:
To make this easy for ourselves and customers, and not change current behaviour, I'd suggest creating a global var for the gem with the original array of error codes to be retried automatically and then we are free to set that and tune it as we like in ZBT/ZAT.

Reply:
Great suggestion, just pushed new config.